test/test_mps.py

```diff
@@ -8449,10 +8439,10 @@ def get_samples():
                 cpu_out = op(*cpu_args, **cpu_kwargs)
                 mps_out = op(*mps_args, **mps_kwargs)
 
-                if op.name == "nn.functional.conv2d" and dtype == torch.float32:
+                if dtype == torch.float32:
                     atol = 1e-4
-                    rtol = 3e-5
```

Comment:
I think we should be using the same values as PyTorch when checking for correctness of fp16/32  (https://pytorch.org/docs/stable/testing.html):

| dtype     | rtol     | atol | 
|---------|-------|------|
| float16  | 1e-3    | 1e-5 |
| float32 | 1.3e-6 | 1e-5 |

Reply:
Reverted the changes for the tolerances. I checked again and there's no new regression with this change.